.changeset/cyan-icons-love.md

```
@@ -0,0 +1,7 @@
+---
+'@mastra/factory': minor
+'@mastra/code-sdk': patch
+'@mastra/core': patch
+---
+
+Added adjustable Build, Plan, and Fast thinking levels to custom model packs.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)